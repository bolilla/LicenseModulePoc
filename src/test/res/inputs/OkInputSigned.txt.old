#This is a comment

#Previous line is empty

   #Previous line is also empty
   
This parameter : has a value


ThisParam:has also a value

#The following lines contain the signature of the file.
#This parameter:has a value
#ThisParam:has also a value
#
===================== SIGNATURE BEGIN =====================
MCwCFBHBCpF9XkU6IPV3Sn8odYce8nk3AhQYB24vRNJR3QiQGSKch2fwLcTPFw==
=====================  SIGNATURE END  =====================
