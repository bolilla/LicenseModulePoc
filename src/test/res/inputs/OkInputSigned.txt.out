#This is a comment

#Previous line is empty

   #Previous line is also empty
   
This parameter : has a value


ThisParam:has also a value

#The following lines contain the signature of the file.
#This parameter:has a value
#ThisParam:has also a value
#
===================== SIGNATURE BEGIN =====================
MCwCFA4DCd5r3QmZGNSOxJ5BtsxXjH6FAhRtN924CCe2+6X9yFBycBpQ9HZKkA==
=====================  SIGNATURE END  =====================
